src/states/node.rs

```diff
@@ -848,6 +945,17 @@ impl Node {
         }
 
         self.add_to_routing_table(public_id, peer_id);
+
+        if let Some(prefix) = self.peer_mgr.routing_table().find_group_prefix(public_id.name()) {
+            let _ = self.send_section_list_signature(prefix, None);
```

Comment:
We don't send signatures _to_ the joining node here - we send the signatures for the updated list of the section where the node joined to _our own_ section. I should probably add a comment here.